tests_page_object_pattern/login_modal.py

```diff
@@ -0,0 +1,17 @@
+from selene import browser, by, be, have
+
+class LoginModal:
+
+    def verify_that_login_modal_opened(self):
+        browser \
+            .element(by.xpath('//input[@type="email"]')).should(be.clickable) \
+            .element(by.xpath('//input[@type="password"]')).should(be.clickable) \
+            .element(by.xpath('//button[@class="button button--large button--green auth-modal__submit ng-star-inserted"]')).should(be.clickable) \
+            .element(by.xpath('//button[@class="auth-modal__register-link button button--link ng-star-inserted"]')).should(be.clickable) \
+            .element(by.xpath('//app-slider/div/button[1]')).should(be.clickable) \
+            .element(by.xpath('//app-slider/div/button[2]')).should(be.clickable)
```

Comment:
for readability purpose, I'd split this into smaller ones:
`browser.element(..).should(...)`
`browser.element(...).should(...)`